client.go

```diff
@@ -229,34 +230,37 @@ func (c Client) handleResponse(req *internalRequest, response *http.Response, in
 	return nil
 }
 
-func contains(slice []int, val int) bool {
-	for _, item := range slice {
-		if item == val {
-			return true
-		}
-	}
-	return false
+// defaultWaitForPendingUpdate check each 50ms the status of a AsyncUpdateID.
+// This method is used for test purpose
+func (c Client) defaultWaitForPendingUpdate(indexUID string, updateID *AsyncUpdateID) (UpdateStatus, error) {
```

Comment:
Since this method is for tests purpose and has nothing to do with the client, can we put it in another file like `test_utils.go` for example? 🙂 Or another name, I don't know the convention in go.

Reply:
My bad, I haven't seen your comment about making it private. I would rather put it in another file because this method is for our only tests.

But If it's a good practice in golang, so, let's do that.

Reply:
@qdequele @eskombro can you confirm?

Reply:
It's more common to have all methods from a struct in the file where the struct is declared. 
But I can refactor it to take the client in param and move it to a test_utils.go file if you want. It's the same ^^

Reply:
I didn't know the good practices about that sorry 😏

Reply:
Ok, let's keep your choice then @alexisvisco.
But the last question: if it it's a private function, how is it possible to call it?

Reply:
I mean, to call it in our tests?

Reply:
All tests are in the same package :)

Reply:
Ok good to know that it works in this case :) thx!